test/integration/patients/patient/patient-flow.js

```diff
@@ -1119,7 +1119,10 @@ context('patient flow page', function() {
       },
     });
 
-    cy.wait('@routeAction');
+    cy
```

Comment:
this prevents the bug.  the other refactors are tested enough within the current tests